modules/client-configuration/pages/tshoot-clients.adoc

```diff
@@ -135,3 +135,32 @@ grains:
       - name.one
       - name.two
 ----
+
+
+
+== SSL Errors on Registration
+
+Some clients experience SSL errors when you register them in the {productname} {webui} using a bootstrap script.
+This can make some operations unusable, including package management and bootstrapping.
+In this case, you will see an error like this:
+
+----
+Repository '<Repository_Name>' is invalid.
```

Comment:
At least from WebUI this is not the error you see (I think you see this from bootstrap script).

Should we include both? Should we include none?

Reply:
What error do we get in the webui? I think for this section including the error is a good idea.

Reply:
```
Rendering SLS 'base:bootstrap' failed: Jinja error: >>> No TLS 1.2 and above for RHEL6 and SLES11. Please check your Apache config. <<< Traceback (most recent call last): File "/usr/lib/python3.6/site-packages/salt/utils/templates.py", line 392, in render_jinja_tmpl output = template.render(**decoded_context) File "/usr/lib/python3.6/site-packages/jinja2/asyncsupport.py", line 76, in render return original_render(self, *args, **kwargs) File "/usr/lib/python3.6/site-packages/jinja2/environment.py", line 1008, in render return self.environment.handle_exception(exc_info, True) File "/usr/lib/python3.6/site-packages/jinja2/environment.py", line 780, in handle_exception reraise(exc_type, exc_value, tb) File "/usr/lib/python3.6/site-packages/jinja2/_compat.py", line 37, in reraise raise value.with_traceback(tb) File "<template>", line 53, in top-level template code File "/usr/lib/python3.6/site-packages/salt/utils/jinja.py", line 211, in jinja_raise raise TemplateError(msg) salt.exceptions.TemplateError: >>> No TLS 1.2 and above for RHEL6 and SLES11. Please check your Apache config. <<< ; line 53 --- [...] {%- if not grains['os_family'] == 'Debian' %} {%- set bootstrap_repo_request = salt['http.query'](bootstrap_repo_url + 'repodata/repomd.xml', status=True, verify_ssl=False) %} {# 901 is a special status code for the TLS issue with RHEL6 and SLE11. #} {%- if bootstrap_repo_request['status'] == 901 %} {{ raise(bootstrap_repo_request['error']) }} <====================== {%- endif %} {%- set bootstrap_repo_exists = (0 < bootstrap_repo_request['status'] < 300) %} bootstrap_repo: file.managed: [...] ---
```
Really ugly, but it's how it is :(

Reply:
Also, the message rendered needs to be updated to include the other products, already mentioned in that PR, so CentOS and Oracle.